src/nidcpower/metadata/functions_addon.py

```diff
@@ -6,7 +6,7 @@
 functions_codegen_method = {
     'InitializeWithChannels':          { 'codegen_method': 'private', 'method_name_for_documentation': '__init__', },
     'InitWithOptions':                 { 'codegen_method': 'no',       },
-    'Initiate':                        { 'codegen_method': 'private', 'public_method_name': 'initiate', },
+    'Initiate':                        { 'codegen_method': 'private', 'method_name_for_documentation': 'initiate', },
```

Comment:
Should be called out in PR description, or moved to its own PR.

Reply:
Made the change in master so that it didn't need to be in each branch